edc-extensions/ssi/ssi-miw-credential-client/src/main/java/org/eclipse/tractusx/edc/iam/ssi/miw/api/MiwApiClientImpl.java

```diff
@@ -137,7 +137,7 @@ private Result<Void> handleVerifyResult(Map<String, Object> response) {
         if (valid) {
             return Result.success();
         } else {
-            return Result.failure(format("Verification failed with response: %s", response));
+            return Result.failure("MIW verification failed");
```

Comment:
Can we provide more info on what failed?

Reply:
should we? I mean, rephrasing it to `"VP verification failed"` is about the extent of info we want to emit, no? if the verification of a VP failed, there is a limited set of possibilities what could've gone wrong anyway. And even if the API did return the exact cause (which I _hope_ it doesn't), we should not leak any details on our side.

Reply:
Ah no you are right, that is returned to the client. We may want to send the failure to the monitor though

Reply:
There is no failure details actually, only valid or not. We can only know if it was the audiance verification that failed or the VP verification